pkgs/development/python-modules/weasyprint/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, isPy3k, fetchPypi, buildPythonPackage
+, pytestrunner
+, cffi, pyphen, cairosvg, html5lib, tinycss2, cairocffi, cssselect2
+, pytestcov, pytest-flake8, pytest-isort }:
+
+buildPythonPackage rec {
+  pname = "weasyprint";
+  version = "44";
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "WeasyPrint";
+    sha256 = "0f5bzwac1c4wgsscdv6d2d08n6g3j7pmj76ja5q31v40vzkz96vr";
+  };
+
+  buildInputs = [
+    pytestrunner
+  ];
+
+  propagatedBuildInputs = [
+    cffi
+    pyphen
+    cairosvg
+    html5lib
+    tinycss2
+    cairocffi
+    cssselect2
+  ];
+
+  # OSError: cannot load library 'libgobject-2.0.so.0': libgobject-2.0.so.0: cannot open shared object file: No such file or directory
```

Comment:
You may need to add `gobject-introspection` to `nativeBuildInputs`. I'm not sure if something like `gtk` is needed in `buildInputs` though.